mappings/net/minecraft/client/render/block/BlockRenderer.mapping

```diff
@@ -1,54 +1,126 @@
 CLASS none/bpr net/minecraft/client/render/block/BlockRenderer
+	CLASS none/bpr$a EnumNeighborInfo
+		FIELD a DOWN Lnone/bpr$a;
+		FIELD b UP Lnone/bpr$a;
+		FIELD c NORTH Lnone/bpr$a;
+		FIELD d SOUTH Lnone/bpr$a;
+		FIELD e WEST Lnone/bpr$a;
+		FIELD f EAST Lnone/bpr$a;
+		FIELD g corners [Lnone/cu;
+		FIELD h shadeWeight F
+		FIELD i doNonCubicWeight Z
+		FIELD n VALUES [Lnone/bpr$a;
+		METHOD <init> (Ljava/lang/String;I[Lnone/cu;FZ[Lnone/bpr$d;[Lnone/bpr$d;[Lnone/bpr$d;[Lnone/bpr$d;)V
+			ARG 0 corners
+			ARG 1 shadeWeight
+			ARG 2 doNonCubicWeight
+		METHOD a getNeighbourInfo (Lnone/cu;)Lnone/bpr$a;
+			ARG 0 facing
 	CLASS none/bpr$b AmbientOcclusionCalculator
+		FIELD b vertexColorMultiplier [F
+		FIELD c vertexBrightness [I
+		METHOD a getAoBrightness (IIII)I
```

Comment:
Ah, this one you've used `Ao` instead of `AO`.
`getAOLightValue` and this both should either be written out or abbreviated I guess.